src/icon_close.js

```diff
@@ -0,0 +1,5 @@
+import { html } from 'lit-element';
+
+export default html`
+<svg height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><circle cx="9" cy="9" fill="#fff" r="9"/><path d="m10 6-3 3.08108108 3 2.91891892" stroke="#333" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"/></g></svg>
+`;
```

Comment:
Do we already have this in `iaux-icons`?

Reply:
No. This is what I mentioned in stand up, and what prompted my discussion with Jim about needing a lot of icons to be properly exported so I can make one update to ia-icons for all of these new components.